src/components/Auth/Register/Register.module.css

```diff
@@ -1,55 +1,90 @@
-/* .container {
-  height: 100%;
-  width: 100%;
-  display: grid;
-  justify-content: center;
-  align-items: center;
+.container {
+	width: 100%;
+	height: 100%;
+	padding: 2em;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	background-image: url('../../src/assets/onboarding 1.png');
 }
 
-.form {
-  display: flex;
-  flex-direction: column;
+header {
```

Comment:
Może zamieńmy to na klasę header i dodajmy w pliku .jsx tę klasę przy header? W razie gdyby miały być u nas inne headery w aplikacje, które mogłyby się nadpisać tymi stylami